ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/proxy/private/deprovision/cucushift-installer-rehearse-ibmcloud-ipi-proxy-private-deprovision-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-proxy-private-deprovision
+  steps: 
+  - ref: gather-installer-boostrap-logs-in-bastion
```

Comment:
Because we did not create provision step, I think we can drop deprovision step here too, and move the steps to workflow yaml file.